package.json

```diff
@@ -18,7 +18,8 @@
     "prisma:seed": "npx prisma db seed --schema=src/db/prisma"
   },
   "prisma": {
-    "seed": "ts-node ./src/db/prisma/seeders/sourceSeeder.ts"
+    "seed": "ts-node ./src/db/prisma/seeders/sourceSeeder.ts",
```

Comment:
You should not add commands in here. Remove them. This is a part of separate task that is assigned to another person.